cypress/e2e/7-workflow-actions.cy.ts

```diff
@@ -197,4 +197,47 @@ describe('Workflow Actions', () => {
 		WorkflowPage.getters.workflowSettingsModal().should('not.exist');
 		WorkflowPage.getters.successToast().should('exist');
 	});
+
+	it('should not be able to delete unsaved workflow', () => {
+		WorkflowPage.getters.workflowMenu().should('be.visible');
+		WorkflowPage.getters.workflowMenu().click();
+		WorkflowPage.getters.workflowMenuItemDelete().closest('li')
+			.should('have.class', 'is-disabled');
+	});
+
+	it('should delete workflow', () => {
+		WorkflowPage.actions.saveWorkflowOnButtonClick();
+		WorkflowPage.getters.workflowMenu().should('be.visible');
+		WorkflowPage.getters.workflowMenu().click();
+		WorkflowPage.getters.workflowMenuItemDelete().click();
+		cy.get('div[role=dialog][aria-modal=true]').should('be.visible');
+		cy.get('button.btn--confirm').should('be.visible');
+		cy.get('button.btn--confirm').click();
+		WorkflowPage.getters.successToast().should('exist');
+		cy.url().should('include', '/workflow/new');
+	});
+
+	it('should duplicate workflow', () => {
+		// Stub window.open so new tab is not getting opened
+		cy.window().then(win => {
+			cy.stub(win, 'open').as('open')
+		})
+
+		WorkflowPage.actions.addNodeToCanvas(MANUAL_TRIGGER_NODE_NAME);
+		WorkflowPage.actions.saveWorkflowOnButtonClick();
+		WorkflowPage.getters.workflowMenu().should('be.visible');
+		WorkflowPage.getters.workflowMenu().click();
+		WorkflowPage.getters.workflowMenuItemDuplicate().click();
+		WorkflowPage.getters.duplicateWorkflowModal().should('be.visible');
+		WorkflowPage.getters.duplicateWorkflowModal().find('input').first().should('be.visible');
+		WorkflowPage.getters.duplicateWorkflowModal().find('input').first().type('{selectall}');
+		WorkflowPage.getters.duplicateWorkflowModal().find('input').first().type(DUPLICATE_WORKFLOW_NAME);
+		WorkflowPage.getters.duplicateWorkflowModal().find('.el-select__tags input').type(DUPLICATE_WORKFLOW_TAG);
+		WorkflowPage.getters.duplicateWorkflowModal().find('.el-select__tags input').type('{enter}');
+		WorkflowPage.getters.duplicateWorkflowModal().find('.el-select__tags input').type('{enter}');
```

Comment:
Can we chain `type` together? (Maybe not possible.)

Reply:
I have noticed that tests are not reliable on CI when chaining. Probably break between each command gives it time to render all elements